test/integration_test/main_test.go

```diff
@@ -96,9 +96,13 @@ func setup() error {
 		return err
 	}
 
-	ci_utils.PxUpgradeHopsURLList = strings.Split(pxUpgradeHopsURLs, ",")
+	if len(pxUpgradeHopsURLs) != 0 {
```

Comment:
`pxUpgradeHopsURLs ` is a string right? do you mean to check the url list length after split? or just if `pxUpgradeHopsURLs == ""`?

Reply:
Done